frontend/src/js/store/organizationSlice/thunks.ts

```diff
@@ -269,6 +279,9 @@ export const getUserOrganization = createAsyncThunk(`${sliceName}/getUserOrganiz
     return Promise.all(tasks);
   });
 });
+export const getUserBilling = createAsyncThunk(`${sliceName}/getUserBilling`, (_, { dispatch }) =>
```

Comment:
```suggestion
export const getUserBillingProfile = createAsyncThunk(`${sliceName}/getUserBilling`, (_, { dispatch }) =>
```
to be aligned with the edit naming..?